doc/releases/index.rst

```diff
@@ -20,22 +20,71 @@ of interest).
 Release Life Cycle and Maintenance
 **********************************
 
-Periodic Releases
-=================
+Major and Minor Release Cadence
+===============================
 
-The Zephyr project provides periodic releases every 4 months leading to the
-long term support releases approximately every 2 years. Periodic and non-LTS
-releases are maintained with updates, bug fixes and security related updates
-for at least two cycles, meaning that the project supports the most recent two
-releases in addition to the most recent LTS.
+The Zephyr Project delivers major releases using a six month cadence roughly timed
+each April and October of the year.
+
+This timescale facilitates regular releases that have strong QA cycles while
+not overwhelming users with too many new releases.
+
+The cadence is predictable and avoids many major holidays in various geographies.
+
+The Zephyr project delivers minor (point) releases on an unscheduled basis and
```

Comment:
"Minor (point) releases" makes it sound like all of 5.x are point releases to "major release" 5, but this doesn't match with the table below, which shows versioning being done the same way as it is today.

IMO having "5.1" be a major and "5.1.x" be a minor release is somewhat confusing since "major" and "minor" are typically used to refer to the first two digits respectively.

Reply:
addressed